helpers/github.ts

```diff
@@ -25,6 +25,8 @@ export const projects = _projects as {
 
 export const DEVPOOL_OWNER_NAME = "ubiquity";
 export const DEVPOOL_REPO_NAME = "devpool-directory";
+export const DEVPOOL_RFC_OWNER_NAME = "ubiquity";
+export const DEVPOOL_RFC_REPO_NAME = "devpool-rfc";
```

Comment:
Perhaps it makes sense to move `DEVPOOL_OWNER_NAME` and `DEVPOOL_REPO_NAME` to env variables in order not to duplicate variables

Reply:
moved all of them to env if that's ok